extensions-core/datasketches/src/main/java/org/apache/druid/query/aggregation/datasketches/quantiles/DoublesSketchComplexMetricSerde.java

```diff
@@ -77,7 +78,7 @@ public Object extractValue(final InputRow inputRow, final String metricName)
           // This corresponds to "A" in base64, so it is not a digit
           if (objectString.isEmpty()) {
             return DoublesSketchOperations.EMPTY_SKETCH;
-          } else if (Character.isDigit(objectString.charAt(0))) {
+          } else if (NumberUtils.isParsable(objectString)) {
```

Comment:
Since this method is going to look at every character of the string to check if it's a number, I wonder if it's better to just try to parse it to a double and then use it if it's not null, maybe
```java
...
          final Double doubleValue;
          if (objectString.isEmpty()) {
            return DoublesSketchOperations.EMPTY_SKETCH;
          } else if ((doubleValue = Doubles.tryParse(objectString)) != null) {
            UpdateDoublesSketch sketch = DoublesSketch.builder().setK(MIN_K).build();
            sketch.update(doubleValue);
            return sketch;
          }
...
```

Reply:
Sounds good to me.